README.md

```diff
@@ -357,7 +357,7 @@ In order to achieve greater coverage and encourage more people to contribute to
         <tr>
             <td><a href="https://en.wikipedia.org/wiki/Linear_search">Linear Search (Recursive)</a></td>
             <td> <!-- C -->
-                <a href="./src/c/BuscaSequencialRecursiva.c">
+                <a href="./src/c/RecursiveSequentialSearch.c">
```

Comment:
Following a similar approach, what do you think about `LinearSearchRecursive`? (I know using `Recursive` as prefix instead of suffix is more correct, but by using it as suffix we make sure all related algorithms, those starting with `LinearSearch`, will be close to each other, when listing the directory files)

Reply:
I like the idea

Reply:
But also with the recursive prefix all recursive algorithms are close

Reply:
But this still seems a little questionable to me, recursive algorithms are together now

Reply:
To be honest, I don't have a strong opinion about that. It's fine to me to keep `Recursive` as prefix 👍